conans/model/build_info.py

```diff
@@ -302,31 +313,67 @@ def __getattr__(self, item):
         return attr
 
     @staticmethod
-    def _merge_lists(seq1, seq2):
-        return seq1 + [s for s in seq2 if s not in seq1]
+    def _merge_lists(seq1, seq2, reverse=False):
```

Comment:
do we need reverse finally?

Reply:
don' know, but this is not `reverse`, it is appending one before the other or vice-versa, but it is not reversing the relative order of the elements.